chat-plugins/info.js

```diff
@@ -109,6 +109,19 @@ var commands = exports.commands = {
 		if ((user === targetUser || user.can('makeroom')) && privaterooms) {
 			buf += '<br />Private rooms: ' + privaterooms;
 		}
+
+		if (user.can('lock') || (!room.isPrivate && user.can('mute', targetUser, room) && targetUser.userid in room.users)) {
+			var bannedFrom = "";
+			for (var i = 0; i < Rooms.global.chatRooms.length; i++) {
+				var thisRoom = Rooms.global.chatRooms[i];
+				if (!thisRoom || thisRoom.isPrivate) continue;
```

Comment:
We may as well allow hidden rooms in this list. Hidden rooms are basically public rooms.